.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,208 @@
+
+      - name: Validate frontend types
+        run: .github/not-on-master.sh docker compose run base yarn typecheck
+
+      - name: Start webknossos, datastore, and tracingstore
+        run: |
+          docker compose up -d webknossos
+          docker compose up -d webknossos-datastore
+          docker compose up -d webknossos-tracingstore
+
+      - name: Run webknossos smoke test
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2034:warning:1:1: i appears unused. Verify use (or export if used externally)

(shellcheck)